lintercat/lintercat.com
# Contact page

### What does this PR do?
Add the contact section. Responsive and with proper code quality,

### Where should the reviewer start?
`/src/components/Contact/Contact.vue`

### Does this add new dependencies? (Y/n)
N

### How should this be manually tested?
1. Build the project
2. Open the root and click the Contact link at the bottom
3. Make sure everything looks and functions properly

### Any background context you want to provide?
N

### What are the relevant issues?
Close #7

### Screenshots / Gifs
![lintercat-contact](https://user-images.githubusercontent.com/5368213/51591882-d7340280-1eb3-11e9-8a7a-d104ad4a06b3.gif)
